ci/dhis2-verify-lib.yml

```diff
@@ -81,35 +82,26 @@ jobs:
     #          with:
     #              node-version: 12.x
     #
-    #        - uses: c-hive/gha-yarn-cache@v1
-    #        - run: yarn install --frozen-lockfile
-    #
     #        - uses: actions/download-artifact@v2
     #          with:
     #              name: lib-build
     #
-    #        - name: Install Cypress binary
-    #          run: yarn cypress install
-    #
     #        - name: End-to-End tests
     #          uses: cypress-io/github-action@v2
     #          with:
-    #              install: false
     #              record: true
     #              parallel: true
-    #              start: ${{ env.SERVER_START_CMD }}
-    #              wait-on: ${{ env.SERVER_URL }}
+    #              # This should be a command that starts the server to test against,
+    #              # without opening a browser.
+    #              start: yarn start:nobrowser
```

Comment:
Just `yarn start` is fine as a default.

Reply:
Good point, I don't think we can assume which command the lib would want to run here. Because of that I would probably say what you have there is fine.

Reply:
Done, I've changed it to just yarn start.